book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] great job. please A method for adding rentals should be present in the book class. [learn more](https://github.com/microverseinc/curriculumruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)
- [ ] kindly ensure the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.